runtime/cli/readme.md

```diff
@@ -89,6 +89,10 @@ the context will be found and replayed sequentially.
 </dd>
 </dl>
 
+> It's recommended to add `.env` to `.gitignore`. Alsa know that credentials to connection will be initially empty. 
```

Comment:
It is recommended to add `.env` to `.gitignore`. Credentials will be empty initially.